Assets/Scripts/Audio/AudioUtils.cs

```diff
@@ -50,7 +50,7 @@ public static GameObject Play2DAudioStream(MP3StreamReader audioStream)
 
 		var audioSource = gameObject.AddComponent<AudioSource>();
 		//audioSource.clip = CreateStreamingAudioClip("tmpAudioClip", audioStream);
-		//audioSource.loop = true;
+		audioSource.loop = true;
```

Comment:
Oh sorry, I actually didn't mean to leave that one in, I was just making some of the debug log warnings go away and forgot to change that one back.